README.md

```diff
@@ -275,14 +275,14 @@ Register index can be cyclic.
 |v0.s4 - v3.s4|VReg4SList|SIMD/Floating point register list containing 32-bit elements x 4 x 4|
 |v0.d2 - v3.d2|VReg2DList|SIMD/Floating point register list containing 64-bit elements x 2 x 4|
 
-You can also use your original instance as mnemonic functions argumetns.
+You can also use your original instance as mnemonic functions arguments.
 Please refer constructor of "C++ class name" in Xbyak_aarch64 files.
 
 ```
 VReg16BList dstList(v0.b16, v3.b16);
 VReg16BList hoge(VReg16B(0), VReg16B(3));
 
-ld4((v0 - v3), ptr(x0)); <--- (1)
+ld4((v0.b16, v3.b16), ptr(x0)); <--- (1)
```

Comment:
Could you change this line to `ld4((v0.b16 - v3.b16), ptr(x0)); <--- (1)`?

The follwing C++ source code
```C++
 0:   VReg16BList dstList(v0.b16, v3.b16);
 1:   VReg16BList hoge(VReg16B(0), VReg16B(3));
 2:   ld4(dstList, ptr(sp));
 3:   ld4(hoge, ptr(sp));
 4:   ld4(v0.b16, ptr(sp));
 5:
 6:   ld4((v0.b16, v1.b16), ptr(sp));
 7:   ld4((v0.b16, v2.b16), ptr(sp));
 8:   ld4((v0.b16, v3.b16), ptr(sp));
 9:
10:   ld4((v0.b16 - v1.b16), ptr(sp));
11:   ld4((v0.b16 - v2.b16), ptr(sp));
12:   ld4((v0.b16 - v3.b16), ptr(sp));
13:
14:   ld4(v0.b16, ptr(sp));
15:
16:   ld4((v0.b16, v1.b16)[0], ptr(sp));
17:   ld4((v0.b16, v2.b16)[0], ptr(sp));
18:   ld4((v0.b16, v3.b16)[0], ptr(sp));
19:
20:   ld4((v0.b16 - v1.b16)[0], ptr(sp));
21:   ld4((v0.b16 - v2.b16)[0], ptr(sp));
22:   ld4((v0.b16 - v3.b16)[0], ptr(sp));
```
 generates the binary 
```
ld4 {v0.16b-v3.16b}, [sp]
ld4 {v0.16b-v3.16b}, [sp]
ld4 {v0.16b-v3.16b}, [sp]

ld4 {v1.16b-v4.16b}, [sp]
ld4 {v2.16b-v5.16b}, [sp]
ld4 {v3.16b-v6.16b}, [sp]

ld4 {v0.16b-v3.16b}, [sp]
ld4 {v0.16b-v3.16b}, [sp]
ld4 {v0.16b-v3.16b}, [sp]

ld4 {v0.16b-v3.16b}, [sp]

ld4 {v1.b-v4.b}[0], [sp]
ld4 {v2.b-v5.b}[0], [sp]
ld4 {v3.b-v6.b}[0], [sp]

ld4 {v0.b-v3.b}[0], [sp]
ld4 {v0.b-v3.b}[0], [sp]
ld4 {v0.b-v3.b}[0], [sp]
```
with the follwing compiler warning.
```
 warning: left operand of comma operator has no effect [-Wunused-value]
   6 |     ld4((v0.b16, v1.b16), ptr(sp));
     |          ~~~^~~
 warning: left operand of comma operator has no effect [-Wunused-value]
   7 |     ld4((v0.b16, v2.b16), ptr(sp));
     |          ~~~^~~
 warning: left operand of comma operator has no effect [-Wunused-value]
   8 |     ld4((v0.b16, v3.b16), ptr(sp));
     |          ~~~^~~
 warning: left operand of comma operator has no effect [-Wunused-value]
  16 |     ld4((v0.b16, v1.b16)[0], ptr(sp));
     |          ~~~^~~
 warning: left operand of comma operator has no effect [-Wunused-value]
  17 |     ld4((v0.b16, v2.b16)[0], ptr(sp));
     |          ~~~^~~
 warning: left operand of comma operator has no effect [-Wunused-value]
  18 |     ld4((v0.b16, v3.b16)[0], ptr(sp));
     |          ~~~^~~
```

In this case, the recommended pattern is `4: ld4(v0.b16, ptr(sp));` and `10: ld4((v0.b16 - v1.b16), ptr(sp));`.

Reply:
Done